modules/http4s-swagger/test/src-ce3/TestCompat.scala

```diff
@@ -28,5 +28,5 @@ trait TestCompat { self: BaseIOSuite =>
   }
 
   def docs(path: String) =
-    Docs[IO](service, path, swaggerUiPath = "swaggerui")
+    Docs.multiple[IO](path, swaggerUiPath = "swaggerui")(service)
```

Comment:
might as well remove the `multiple` method altogether and just document the new way of wiring things up 😄 



Also we might want to make the `swaggerUiPath` configurable in the PartiallyApplied construct

Reply:
> Also we might want to make the swaggerUiPath configurable in the PartiallyApplied construct

I started doing it, but I found that cumbersome. I figured, if someone needs to go all the way down and customize the swagger ui, they can use `Docs.build[IO](path, swaggerUiPath = "swaggerui")(service)` like in this `TestCompat`

note `build` is the new name I used instead of multiples`

-----

will update the docs, but not much has changed from a user point of view

Reply:
I can do that